apps/judicial-system/backend/src/app/modules/case/filters/test/appealsCourtUserFilter.spec.ts

```diff
@@ -63,7 +64,12 @@ describe.each(courtOfAppealsRoles)('appeals court user %s', (role) => {
       describe.each(accessibleCaseAppealStates)(
         'accessible case appeal state %s',
         (appealState) => {
-          const theCase = { type, state, appealState } as Case
+          const theCase = {
+            type,
+            state,
+            appealState,
+            appealReceivedByCourtDate: new Date(),
```

Comment:
I believe the convention in the backend is to use nowFactory for creating dates.. Consider doing that..

Reply:
Done